support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -0,0 +1,201 @@
+`;
+
+const buttonOverrides = css`
+	min-height: unset;
+	height: unset;
+	text-decoration: none;
+	${textSans.xsmall()};
+	color: ${palette.neutral[20]};
+
+	.src-button-space {
+		width: ${space[1]}px;
+	}
+`;
+
+const iconCss = (flip: boolean) => css`
+	svg {
+		max-width: ${space[4]}px;
+		transition: transform 0.3s ease-in-out;
+
+		${flip && 'transform: rotate(180deg);'}
+	}
+`;
+
+const checklistContainer = css`
+	margin-top: ${space[5]}px;
+`;
+
+const detailsSection = css`
+	display: flex;
+	flex-direction: column;
+	margin-bottom: ${space[6]}px;
+`;
+
+const termsAndConditions = css`
+	${textSans.xxsmall()}
+	color: #606060;
+
+	p {
+		margin-top: ${space[1]}px;
+	}
+`;
+
+export type ContributionsOrderSummaryProps = {
+	contributionType: ContributionType;
+	total: string;
+	checkListData: CheckListData[];
+	onAccordionClick?: (opening: boolean) => void;
+	headerButton?: React.ReactNode;
+	tsAndCs?: React.ReactNode;
+};
+
+const supportTypes = {
+	ONE_OFF: 'Single',
+	MONTHLY: 'Monthly',
+	ANNUAL: 'Annual',
+};
+
+const timePeriods = {
+	MONTHLY: 'month',
+	ANNUAL: 'year',
+};
+
+function totalWithFrequency(total: string, contributionType: ContributionType) {
+	if (contributionType === 'ONE_OFF') {
+		return total;
+	}
+	return `${total}/${timePeriods[contributionType]}`;
+}
+
+export function ContributionsOrderSummary({
+	contributionType,
+	total,
+	checkListData,
+	onAccordionClick,
+	headerButton,
+	tsAndCs,
+}: ContributionsOrderSummaryProps): JSX.Element {
+	const [showDetails, setShowDetails] = useState(false);
+
+	const showAccordion =
+		contributionType !== 'ONE_OFF' && checkListData.length > 0;
+
+	return (
+		<div css={componentStyles}>
+			<div css={[summaryRow, rowSpacing, headingRow]}>
+				<h2 css={heading}>Your support</h2>
+				{headerButton}
+			</div>
+			<hr css={hrCss} />
+			<div css={[rowSpacing, detailsSection]}>
+				<div css={summaryRow}>
+					<p>{supportTypes[contributionType]} support</p>
+					{showAccordion && (
+						<Button
+							priority="subdued"
+							aria-expanded={showDetails ? 'true' : 'false'}
+							onClick={() => {
+								onAccordionClick?.(!showDetails);
+								setShowDetails(!showDetails);
+							}}
+							icon={<SvgChevronDownSingle />}
+							iconSide="right"
+							cssOverrides={[buttonOverrides, iconCss(showDetails)]}
+						>
+							{showDetails ? 'Hide details' : 'View details'}
+						</Button>
+					)}
+				</div>
+				{showAccordion && showDetails && (
+					<div css={checklistContainer}>
+						<CheckmarkList
+							checkListData={checkListData}
+							style="compact"
+							iconColor={palette.brand[500]}
+						/>
+					</div>
+				)}
+			</div>
+			<hr css={hrCss} />
+			<div css={[summaryRow, rowSpacing, boldText, totalRow(!!tsAndCs)]}>
+				<p>Total</p>
+				<p>{totalWithFrequency(total, contributionType)}</p>
+			</div>
+			{tsAndCs ? (
+				<div css={termsAndConditions}>
+					<hr css={hrCss} />
+					{tsAndCs}
+				</div>
+			) : (
+				<></>
+			)}
```

Comment:
could this become:

```
{!!tsAndCs && (
  <div css={termsAndConditions}>
    <hr css={hrCss} />
    {tsAndCs}
  </div>
)}
``` 

Falsey in the left hand side of the `&&` operator would mean that react doesn't render anything and therefore no need for the ternary (sorry for the pedantry)